nanoFramework.Graphics/Input/ButtonEnum.cs

```diff
@@ -7,371 +7,819 @@
 namespace nanoFramework.UI.Input
 {
     /// <summary>
-    /// 
+    /// The button or key.
     /// </summary>
     public enum Button
     {
-        /// <summary>  </summary>
+        /// <summary>None.</summary>
         None = 0,
-        /// <summary>  </summary>
+
+        /* Those are the mouse keys 0x01 -> 0x06 */
```

Comment:
Do we really need mouse keys in nano?? 🤔

Reply:
yes, we should keep that. Those are the standard key used everywhere. For HID implementation for example, those are the ones you can map things.

Reply:
> For a future discussion: a general clean-up is due for things like mouse events and keys, keyboard with function keys, numeric keypad, sessions, etc. All this is bagagge from the NETMF days intended for usage scenarious that we are not (nor planning) to address.

And for those ones, we definitely should KEEP them. You can make a mouse out of nano if you want with a screen attached, why not :-) Again, the codes are HID ones, they are shared across any platform.